content/workspace/openbb-copilot.md

```diff
@@ -12,336 +12,616 @@ keywords:
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/0a5e0075-7c50-44cb-8d45-2be1817deab5.png"
+    width="100%"
+  />
+</div>
+
+## Basics
 
-To use OpenBB Copilot, click on the purple icon located at the center right of the
-Workspace screen:
+### Structure
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/open_copilot.jpg" alt="open openbb copilot" width="70%" height="70%" />
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/d7efdb97-fecd-4d58-8094-84fa82bf4bd4.png"
+    width="40%"
+  />
 </div>
 
-Under most circumstances, you can treat OpenBB Copilot like a personal research
-assistant. OpenBB Copilot has access to the widgets (including your uploaded
-files) on your currently-active dashboard, the OpenBB API and any custom backend
-endpoints you have added to OpenBB Workspace. 
+The Copilot interface is designed for intuitive interaction with a clean, three-section layout that maximizes usability and clarity.
+
+- **Header:** Contains essential controls for conversation management, including the current conversation the user is in. But also the capability to start a new chat, clear the current chat history, expand to full AI-mode or hide the agent.
+
+- **Body:** The primary interaction area where conversations unfold. This section displays the Copilot's responses, step-by-step reasoning processes, generated artifacts (charts, tables, code), and any intermediate results. The body automatically scrolls and organizes content chronologically, making it easy to follow complex multi-step analyses. When hovering on messages it also displays the datetime associated with these.
+
+- **Footer:** This is where the data and widget management occurs for what will be used as context for the agent. This is also where the user can add their own custom AI agents or access their prompt library.
+
+### Full AI-chat mode
 
-By utilizing all of these sources of data, OpenBB Copilot can assist you in
-performing a wide variety of research and analysis tasks.
+The Copilot panel offers adaptive sizing to match your workflow needs.
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.us-east-1.amazonaws.com/docs/copilot/ada_opened.png" alt="opened openbb copilot" width="70%" height="70%" />
+In side-panel mode, it provides quick access for simple queries while maintaining your primary workspace view.
+
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/1f9ba730-2516-4eac-b7c9-ca6e9382ba78.png"
+    width="100%"
+  />
 </div>
 
-## Understanding Chats
+When expanded to full-screen mode, you gain maximum real estate for complex conversations, detailed reasoning steps, and large artifacts like comprehensive charts or extensive data tables.
```

Comment:
Ah, you see. You're using "full screen" not "full ai" here. I think "full screen" makes a lot more sense and is easier to understand.